test/helpers/download-test-browsers.js

```diff
@@ -2,18 +2,37 @@
 
 const seleniumAssistant = require('selenium-assistant');
 
+const MAX_RETRIES = 3;
 let forceDownload = false;
 if (process.env.TRAVIS) {
   forceDownload = true;
 }
 
+const downloadBrowser = (name, version, attempt) => {
```

Comment:
Nit: set default value of `attempt` to 0, so we don't need to pass it in the first call